### DISCUSSION TITLE
Silent install and uninstall commands

### DISCUSSION THREAD

**Discussion by SeamusW92** [brackets-cont/brackets#324] [Q&A] [answered]
Hi All, 

I am looking to deploy this software to some of our Pupils who are on intune devices, could you help me out with the silent install and uninstall instructions at all? this would be much appreciated. 

Thanks in advance.

**Comment by abose** [comment]
@SeamusW92 We recommend using the new builds available at https://phcode.io  which is the next generation of Brackets.

For phoenix code, download the installer from the above URL and, the silent install command is: 

```bash
c:\downloads\> phoenix_installer.exe /P
```

**Comment by HUguyBB** [comment]
This did not work for me. In the Software Center, it just stays in the "Installing" stage. Are there other switches that need to be added?

**Reply by abose** [reply, marked as answer]
Try the `phoenix_installer.exe /S` option

**Reply by HUguyBB** [reply]
Thanks! Software Center said it installed, which is a step in the right direction. Unfortunately, I can't find it in any of the program folders. I did locate the .exe file in the C:\Windows\ccmcache folder, where Software Center downloads files first before installation. Wondering why it's not trying to install it.

**Reply by abose** [reply]
Phoenix Code is installed in the `%appdata%` folder. It should also be searchable from the start menu.

**Reply by neddington** [reply]
@abose It installs to the app data of a user which would indicate its installed on a per-user rather than per-machine. Brackets.io used to come in an MSI that could be deployed out to machines and installed per-machine (eg. Installed to C:\Program Files\Brackets\ ). Phoenix Code only comes in an exe and appears to be a per-user installer (eg. Installs to C:\Users\Username\Appdata ) which isn't available to all users and is only available to the user who installed the software. That would seem to limit this software from being able to be mass deployed to machines via systems like SCCM, Intune, Action1, PDQ, PMP, etc. 

Are there plans to eventually include a per-machine installer? Otherwise, there unfortunately isn't a way to mass deploy this software and use it in an enterprise environment as by default for security purposes most block non-admins from being able to run msi and exe's even when the install in a user context instead of a system context. As well as some deployment systems just do not support deployment of per-user installers.

**Reply by abose** [reply]
@neddington a system wide installer is possible and we experimented it with before, but decided to stick with user level installer for general eaze of use.

In the next release, there is a plan to vend system wide installers too.
